buildkite/scripts/dump-mina-type-shapes.sh

```diff
@@ -20,3 +20,5 @@ export TYPE_SHAPE_FILE=${MINA_COMMIT_SHA1}-type_shape.txt
 
 echo "--- Create type shapes git note for commit: ${MINA_COMMIT_SHA1}"
 mina internal dump-type-shapes > ${TYPE_SHAPE_FILE}
+
+source buildkite/scripts/gsutil-upload.sh ${TYPE_SHAPE_FILE} gs://mina-type-shapes
```

Comment:
Why is this now necessary?

Reply:
We did an effort recently to setup  agents in a way that now  env vars which allows to correctly setup gsutil are passed to toolchain and allow us to upload file to gs directly from toolchain. Previously we had to ran gsutil directly from agent which was a bit of dance (dumping file on toolchains, then uploading on agents level).

In `VersionLinter.dhall` i removed line with upload to gcloud bucket